plugins/chat/spec/models/chat/direct_message_channel_spec.rb

```diff
@@ -110,4 +109,16 @@
       expect(channel.slug).to eq(nil)
     end
   end
+
+  describe "#direct_message_group?" do
+    it "returns false if the DirectMessage chatable is not for a group DM" do
+      channel.chatable.update!(group: false)
+      expect(channel.direct_message_group?).to eq(false)
+    end
+
+    it "returns true if the DirectMessage chatable is for a group DM" do
+      channel.chatable.update!(group: true)
+      expect(channel.direct_message_group?).to eq(true)
+    end
+  end
```

Comment:
Potential to use RSpec matching methods:

```suggestion
  describe "#direct_message_group?" do
    it "returns false if the DirectMessage chatable is not for a group DM" do
      channel.chatable.update!(group: false)
      expect(channel).to be_direct_message_group
    end

    it "returns true if the DirectMessage chatable is for a group DM" do
      channel.chatable.update!(group: true)
      expect(channel).not_to be_direct_message_group
    end
  end
```

Reply:
Sure, will do